sklego/notinstalled.py

```diff
@@ -28,7 +31,7 @@ def __init__(self, package_name: str, version: str = None):
                 + f"`python -m pip install scikit-lego[{extra_name}]` or "
                 + "`python -m pip install scikit-lego[all]`. "
                 + "For more information, check the 'Dependency installs' section of the installation docs at "
-                + "https://scikit-lego.readthedocs.io/en/latest/install.html"
+                + "https://scikit-lego.netlify.app/install"
```

Comment:
Unrelated but was pointing to the wrong page

Reply:
Once we move everything unto mkdocs ... I think we'll also port away from netlify and move it all to Github Pages. That deployment is super straightforward.